runtime/src/hardened_unpack.rs

```diff
@@ -199,6 +205,76 @@ where
     }
 }
 
+// return Err on file system error
+// return Some(path) if path is good
+// return None if we should skip this file
+fn sanitize_path(entry_path: &Path, dst: &Path) -> Result<Option<PathBuf>> {
+    // We cannot call unpack_in because it errors if we try to use 2 account paths.
```

Comment:
It would be helpful to call out and justify differences between our implementation and unpack_in, perhaps in code below?